sdk/core/azure-core-tracing-opencensus/src/main/java/com/azure/core/tracing/opencensus/OpenCensusTracer.java

```diff
@@ -181,6 +181,14 @@ public Context extractContext(String diagnosticId, Context context) {
         return AmqpPropagationFormatUtil.extractContext(diagnosticId, context);
     }
 
+    @Override
+    public Context getSharedSpanBuilder(String spanName, Context context) {
+        logger.logExceptionAsError(
```

Comment:
You'll have to throw this so you don't need to return context.none

Reply:
I think we get some error saying you can't directly throw you should use a logger to do that.